src/views/v2/emails/template-parts/body/footer.php

```diff
@@ -20,7 +20,7 @@
  * @var string $header_text_color Header text color.
  */
 
-if ( empty( $footer_content ) || empty( $footer_credit ) ) {
+if ( empty( $footer_content ) && empty( $footer_credit ) ) {
```

Comment:
Even though, if any is empty, it should bail. Have you tested that's fixing the issue?

Reply:
I mean, if the content is empty OR the footer credit is set to OFF, it should bail.

Reply:
And, yes. I tested this and it fixes the issue.

Reply:
Ahh, right! I thought the footer content was the content of the credit. The footer content is actually what's coming from the setting. Thanks for the clarification.